src/Components/Charts/Pie1/Pie1.tsx

```diff
@@ -1,54 +1,79 @@
-import React, { useContext, useEffect, useState } from "react";
+import React, { useContext, useEffect, useState} from "react";
 import { ResponsiveBar } from '@nivo/bar'
 import axios from "axios";
-import { AuthContext } from "../../Context/AuthContext";
 
-import { ResponsiveStream } from '@nivo/stream'
-export default function Bar() {
+
+
+import { PieChart, Pie, Cell } from "recharts";
+import { AuthContext } from "../../../Context/AuthContext";
+
+export default function Pie1() {
+
   const { baseUrl, requstHeaders }: any = useContext(AuthContext);
-  const [contTasks, setContTasks] = useState([]);
-  const [contUsers, setcontUsers] = useState([]);
-  const getContTasks = () => {
+  const [tasksCount, setTasksCount] = useState({});
+ 
+  
+  const getTasksCount = () => {
     axios
       .get(`${baseUrl}/Task/count`, {
         headers: requstHeaders,
       })
       .then((response) => {
-        console.log(response);
-        setContTasks(response?.data);
+     
+        setTasksCount(response?.data);
       })
       .catch((error) => {
         console.log(error);
       });
   };
 
-  const getContUsers = () => {
-    axios
-      .get(`${baseUrl}/Users/count`, {
-        headers: requstHeaders,
-      })
-      .then((response) => {
-        console.log(response);
-        setcontUsers(response?.data);
-      })
-      .catch((error) => {
-        console.log(error);
-      });
-  };
+
+ 
   useEffect(() => {
-    getContTasks();
-    getContUsers();
+    getTasksCount();
+    
   }, []);
-  const data = [
-    {  Month: 1,toDo: 5, inProgress: 6,done:10 },
-    {  Month: 2, toDo: 7, inProgress: 2,done:15 },
-    { Month: 3, toDo: 3, inProgress: 5,done:14 },
-    { Month: 4, toDo: 4, inProgress: 3,done:9 },
+//   const {toDo,inprogress,done} = 
+//     tasksCount
+//   ;
+
+const data = [
+    { name: "ToDo", value: tasksCount.toDo },
+    { name: "inProgress", value: tasksCount.inProgress},
+    { name: "Done", value: tasksCount.done},
+   
   ];
+
+ 
+   
+  
+  const COLORS = ["#0088FE", "#b72e89", "#956c14", ];
   return (
     <>
       <div className="chart">
-      <ResponsiveBar
+
+      <PieChart width={250} height={400}>
```

Comment:
you need to look more into the documentation of the package.
As the Pie chart is not interactive nor it shows the percentage of labels.
It just shows a blue circle with no indication.
![Uploading image.png…]()